lib/bemhtml/index.js

```diff
@@ -158,8 +158,9 @@ BEMHTML.prototype.renderClose = function renderClose(prefix,
 
   // NOTE: maybe we need to make an array for quicker serialization
   attrs = utils.extend(attrs, ctx.attrs);
-  if (attrs) {
-    var name; // TODO: do something with OmetaJS and YUI Compressor
+  if (attrs && typeof attrs === 'object' && !Array.isArray(attrs) &&
+      attrs !== null) {
+    var name;
     /* jshint forin : false */
     for (name in attrs) {
```

Comment:
should be `for (var name in attrs) {`

also, as I remember, there was one more similar TODO somewhere